cpp/plugins/cucim.kit.cuslide/src/cuslide/jpeg/libjpeg_turbo.cpp

```diff
@@ -41,6 +45,12 @@ struct my_error_mgr
     boolean warning, stopOnWarning;
 };
 
+enum
```

Comment:
Should we name this `enum`?

Reply:
Ok that's fine. See we already have the license in 3rd party. Should we add a note to this file about this code coming from there?

Reply:
The information is in lines 18-26 of this file
```
/**
 * Code below is derived from the libjpeg-turbo's code which is under three compatible
 * BSD-style open source licenses
 * - The IJG (Independent JPEG Group) License
 * - The Modified (3-clause) BSD License
 * - The zlib License
 * Please see LICENSE-3rdparty.md for the detail.
 *  - https://github.com/libjpeg-turbo/libjpeg-turbo/blob/00607ec260efa4cfe10f9b36d6e3d3590ae92d79/tjexample.c
 *  - https://github.com/libjpeg-turbo/libjpeg-turbo/blob/2.0.6/turbojpeg.c#L1241
 */
 ```
And in LICENSE-3rdparty.md,

```
libjpeg-turbo
- This software is based in part on the work of the Independent JPEG Group.
- License: libjpeg-turbo is covered by three compatible BSD-style open source licenses
  - The IJG (Independent JPEG Group) License
  - The Modified (3-clause) BSD License
  - The zlib License
  - https://github.com/libjpeg-turbo/libjpeg-turbo/blob/master/LICENSE.md
- Copyright:
  - D. R. Commander
  - Viktor Szathmáry
- Files:
  - cpp/plugins/cucim.kit.cuslide/src/cuslide/jpeg/libjpeg_turbo.cpp : Implementation of jpeg decoder.
```

Do you think additional information is needed here?

Reply:
No that seems sufficient